tests/e2e/util.go

```diff
@@ -316,6 +320,75 @@ func createPVC(client clientset.Interface, pvcnamespace string, pvclaimlabels ma
+		pvclaims = append(pvclaims, pvclaim)
+	}
+	for index := 1; index <= worker/2; index++ {
+		pvcspec := getPersistentVolumeClaimSpecWithStorageClass(pvcnamespace, ds, storageclass, pvclaimlabels, accessMode)
+		ginkgo.By(fmt.Sprintf("Creating PVC using the Storage Class %s with disk size %s and labels: %+v accessMode: %+v", storageclass.Name, ds, pvclaimlabels, accessMode))
+		pvclaim, err := fpv.CreatePVC(client, pvcnamespace, pvcspec)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), fmt.Sprintf("Failed to create pvc with err: %v", err))
+		pvclaims = append(pvclaims, pvclaim)
+	}
+}
+
+// scaleCreateDeletePVC helps create and delete pvc with given namespace and labels using given storage class
+func scaleCreateDeletePVC(client clientset.Interface, pvcnamespace string, pvclaimlabels map[string]string, ds string, storageclass *storagev1.StorageClass, accessMode v1.PersistentVolumeAccessMode, wg *sync.WaitGroup, lock *sync.Mutex) {
+	worker := GetAndExpectIntEnvVar(envWorkerPerRoutine)
+	ctx, cancel := context.WithCancel(context.Background())
+	var totalPVCDeleted int = 0
+	defer cancel()
+	defer wg.Done()
+	for index := 1; index <= worker; index++ {
+		pvcspec := getPersistentVolumeClaimSpecWithStorageClass(pvcnamespace, ds, storageclass, pvclaimlabels, accessMode)
+		ginkgo.By(fmt.Sprintf("Creating PVC using the Storage Class %s with disk size %s and labels: %+v accessMode: %+v", storageclass.Name, ds, pvclaimlabels, accessMode))
+		pvclaim, err := fpv.CreatePVC(client, pvcnamespace, pvcspec)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), fmt.Sprintf("Failed to create pvc with err: %v", err))
+
+		lock.Lock()
+		pvclaims = append(pvclaims, pvclaim)
+		pvclaimToDelete := randomPickPVC(index)
+		lock.Unlock()
+
+		//check if volume is present or not
+		pvclaimToDelete, err = client.CoreV1().PersistentVolumeClaims(pvclaimToDelete.Namespace).Get(ctx, pvclaimToDelete.Name, metav1.GetOptions{})
+
+		if err == nil {
+			// Waiting for PVC to be bound
+			pvclaimsToDelete := append(pvclaimsToDelete, pvclaimToDelete)
+			_, err = fpv.WaitForPVClaimBoundPhase(client, pvclaimsToDelete, framework.ClaimProvisionTimeout)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			err = fpv.DeletePersistentVolumeClaim(client, pvclaimToDelete.Name, pvcnamespace)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			totalPVCDeleted++
+		}
+	}
+	framework.Logf("Total number of deleted PVCs %v", totalPVCDeleted)
+}
+
+func randomPickPVC(pvcIndex int) *v1.PersistentVolumeClaim {
+	var pvclaimToDelete *v1.PersistentVolumeClaim
+	index := rand.Intn(len(pvclaims))
+	newpvclaims := make([]*v1.PersistentVolumeClaim, 0)
+	for i := 0; i < len(pvclaims); i++ {
+		if i != index {
+			newpvclaims = append(newpvclaims, pvclaims[i])
+		}
+	}
+	pvclaimToDelete = pvclaims[index]
+	pvclaims = newpvclaims
+	framework.Logf("pvc to delete %v", pvclaimToDelete.Name)
+	return pvclaimToDelete
```

Comment:
let's keep it simple, looping through the list is not needed

```golang
newpvclaims := []*v1.PersistentVolumeClaim{}
pvclaims[len(pvclaims)-1], pvclaims[index] = pvclaims[index] , pvclaims[len(pvclaims)-1]
pvclaimToDelete := pvclaims[len(pvclaims)-1]
pvclaims = pvclaims[:len(pvclaims)-1]
return pvclaimToDelete
```

Reply:
Added the same, Thanks @sashrith